pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -590,6 +556,22 @@ func TestGetTaskFunc_RemoteResolution(t *testing.T) {
 			taskYAMLStringWithBetaFeatures,
 		}, "\n"),
 		wantErr: true,
+	}, {
+		name: "v1beta1 cluster task",
+		taskYAML: strings.Join([]string{
+			"kind: ClusterTask",
+			"apiVersion: tekton.dev/v1beta1",
+			taskYAMLString,
+		}, "\n"),
+		wantTask: parse.MustParseV1beta1Task(t, taskYAMLString),
+	}, {
+		name: "v1beta1 task without defaults",
```

Comment:
is this meant to replace the "remote-task-without-defaults"? would you mind adding a bit about the reason for these changes to your commit message?

Reply:
yes:) thanks actually I think this test case could also be removed. the cases are already covered